### PR TITLE
*: reorganize code

### PR DESCRIPTION
**Pull request by BusyJay**

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

**Comment by ngaut** [comment]
PTAL @siddontang

File: Cargo.toml [hunk 1]
```diff
@@ -26,3 +26,6 @@ path = "test/test.rs"
 [dependencies]
 libc = "0.1.8"
 tempdir = "0.3.4"
+
+[dependencies.librocksdb_sys]
+path = "librocksdb_sys"
```
Comment (siddontang) [inline Cargo.toml]: why use this instead of ffi directly?

Comment (BusyJay) [inline Cargo.toml]: When we want to do something like static link, we may create a build.rs to control the compilation progress. Make the code releated to ffi as a subproject make things more clear and clean.

**Comment by siddontang** [comment]
LGTM

**Pull request merged**